packages/frontend/components/scaffold-eth/Balance.tsx

```diff
@@ -0,0 +1,81 @@
+import { BigNumberish, ethers } from "ethers";
+import { useEffect, useState } from "react";
+import { useBalance } from "wagmi";
+
+type BalanceProps = {
+  address: string;
+  price: number;
+  wrapperClasses?: string;
+};
+
+/**
+ Display balance of an address
+*/
+
+export default function Balance({ address, price, wrapperClasses }: BalanceProps) {
+  const [isEthBalance, setIsEthBalance] = useState(true);
+  const [balance, setBalance] = useState<any>();
+
+  const {
+    data: fetchBalanceData,
+    isError,
+    isLoading,
+  } = useBalance({
+    addressOrName: address,
+    watch: true,
+    cacheTime: 5_000,
```

Comment:
You can add a ToDo here (or even better, create an issue) to talk about this value.

This shouldn't be used if we are in the local chain (so we get the value updated super fast when prototyping on localhost). For other networks, we should a sensible default (like 30 secs) in a config file, where people can change it.